thread_sync.c

```diff
@@ -836,18 +836,48 @@ queue_closed_result(VALUE self, struct rb_queue *q)
  *
  */
 
+static ID id_each;
```

Comment:
Predefined ID is available.

```suggestion
#define id_each idEach
```